docs/api/sandbox-option.md

```diff
@@ -166,7 +166,7 @@ Currently the `require` function provided in the preload scope exposes the
 following modules:
 
 - `child_process`
-- `electron` (crashReporter, remote and ipcRenderer)
+- `electron` (crashReporter, remote, ipcRenderer, and webFrame)
```

Comment:
Maybe this is a good time to add the supported modules as a sub-list now that it is starting to get long.

```
- `electron`
  - `crashReporter`
  - `ipcRenderer`
  - `remote`
  - `webFrame`
```

Which will render like:

- `electron`
  - `crashReporter`
  - `ipcRenderer`
  - `remote`
  - `webFrame`